src/schema-builder/SchemaBuilder.ts

```diff
@@ -95,19 +95,23 @@ export class SchemaBuilder {
     // Private Methods
     // -------------------------------------------------------------------------
 
+    protected get entityToSyncMetadatas(): EntityMetadataCollection {
+        return <EntityMetadataCollection>this.entityMetadatas.filter( metadata => !metadata.table.skipSchemaSync );
```

Comment:
can you reformat it this way: `<EntityMetadataCollection> this.entityMetadatas.filter(metadata => !metadata.table.skipSchemaSync)`